realionetwork_3300-4/genesis.json

```diff
@@ -0,0 +1,447 @@
+          "min_gas_price": "0.000000000000000000",
+          "min_gas_multiplier": "0.500000000000000000"
+        },
+        "block_gas": "0"
+      },
+      "genutil": {
+        "gen_txs": [
+          {
+            "body": {
+              "messages": [
+                {
+                  "@type": "/cosmos.staking.v1beta1.MsgCreateValidator",
+                  "description": {
+                    "moniker": "test",
+                    "identity": "",
+                    "website": "",
+                    "security_contact": "",
+                    "details": ""
+                  },
+                  "commission": {
+                    "rate": "0.100000000000000000",
+                    "max_rate": "0.200000000000000000",
+                    "max_change_rate": "0.010000000000000000"
+                  },
+                  "min_self_delegation": "1",
+                  "delegator_address": "",
+                  "validator_address": "realiovaloper1jyrr9ga485mzdw6u7w7vcvcmhz8h6zq86p0un6",
+                  "pubkey": {
+                    "@type": "/cosmos.crypto.ed25519.PubKey",
+                    "key": "uUAhL00RoiPRfc53yhrkTCQ4XQh9mOwmnnUqek77QMY="
+                  },
+                  "value": {
+                    "denom": "ario",
+                    "amount": "10000000000000000000"
+                  }
+                }
+              ],
+              "memo": "472a4b1d01db8b0c53d0ff722062c4c75fa85c82@192.168.1.86:26656",
+              "timeout_height": "0",
+              "extension_options": [],
+              "non_critical_extension_options": []
+            },
+            "auth_info": {
+              "signer_infos": [
+                {
+                  "public_key": {
+                    "@type": "/os.crypto.v1.ethsecp256k1.PubKey",
+                    "key": "Ar0JllIA69c2U5Nm2bOODWJ0uTU5pgQ85qKlQP7sWXRV"
+                  },
+                  "mode_info": {
+                    "single": {
+                      "mode": "SIGN_MODE_DIRECT"
+                    }
+                  },
+                  "sequence": "0"
+                }
+              ],
+              "fee": {
+                "amount": [],
+                "gas_limit": "200000",
+                "payer": "",
+                "granter": ""
+              },
+              "tip": null
+            },
+            "signatures": [
+              "P5VZ/pBrlVv/31IsQh4/9zBmbFOccfpIjoFt8NpBvFQsy/oi9s25PtFSKKF/53hq0wlCtxC5c9lvQBFWCpTq3wA="
+            ]
+          }
+        ]
+      },
+      "gov": {
+        "starting_proposal_id": "1",
+        "deposits": [],
+        "votes": [],
+        "proposals": [],
+        "deposit_params": {
+          "min_deposit": [
+            {
+              "denom": "ario"
+            }
+          ]
+        },
+        "voting_params": null,
+        "tally_params": null,
+        "params": {
+          "min_deposit": [
+            {
+              "denom": "stake",
```

Comment:
@catShaark is this correct? seems like overlap with line 261

Reply:
This is the right format of the gov module genesis, we just need to change ario-> stake